allauth/headless/mfa/views.py

```diff
@@ -44,14 +45,21 @@ def get(self, request, *args, **kwargs):
         return response.AuthenticatorsResponse(request, authenticators)
 
 
+# TODO: Enforce reauthentication
```

Comment:
@pennersr BTW, I noticed that in headless mode, enabling MFA does not explicitly check the user has authenticated recently. Is that done intentionally? Is the idea that headless allauth applications should enforce that via a separate API call, i.e. to `headless:browser:mfa:reauthenticate`?

Reply:
When activating TOTP, headless invokes `flows.totp.activate_totp()`, which has `raise_if_reauthentication_required(request)`. Deleting TOTP, viewing and regenerating recovery codes is handled similarly. Doesn't that cover this?

Reply:
Hmmm I will play around and confirm!